presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -100,6 +102,14 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
         return tableStatistics.build();
     }
 
+    private Set<String> filterPartitionColumnFrom(Map<String, ColumnHandle> tableColumns, List<HivePartition> hivePartitions)
+    {
+        return tableColumns.entrySet().stream()
+                .filter(e -> !hivePartitions.get(0).getKeys().containsKey(e.getValue()))
```

Comment:
`s/e/entry`

Stupid question. Can `hivePartitions.get(0).getKeys()` be different than `hivePartitions.get(1).getKeys()`?

Reply:
Just drop `hivePartitions` arg.
Cast `ColumnHandle` to `HiveColumnHandle`.
And use `HiveColumnHandle.getType()` to determine if column is partitioning column.